docs/usage/models.md

```diff
@@ -424,6 +424,21 @@ Where `Field` refers to the [field function](schema.md#field-customisation).
 Here `a`, `b` and `c` are all required. However, use of the ellipses in `b` will not work well
 with [mypy](mypy.md), and as of **v1.0** should be avoided in most cases.
 
+If you want to specify a field that can take a `None` value while still being required, you can use `Optional` with `...`:
+
+```py
+from typing import Optional
+from pydantic import BaseModel, Field
+
+class Model(BaseModel):
+    a: Optional[int]
+    b: Optional[int] = ...
+    c: Optional[int] = Field(...)
```

Comment:
```suggestion
    c: Optional[int] = Field(...)

print(Model(b=1, c=2))

try:
    Model(a=1, b=2)
except ValidationError as e:
    print(e)
```

Probably useful to have one example of this

Reply:
Perfect. I hadn't seen the whole setup to run files and show the results inline. Very clever. :rocket: